csrc/scheduler/reduction_heuristic.h

```diff
@@ -133,8 +135,15 @@ class ReductionParams : public HeuristicParams {
   // block_dim_inner_reduction_extra (usually TIDy)
   ParallelType block_dim_inner_reduction_extra = ParallelType::Serial;
 
-  // use shared memory for persistent buffer, if false, will use registers
+  // Use shared memory for persistent buffer, if false, will use registers.
```

Comment:
nitpick: We might want update this comment, as well as the variable name.

The old logic, iiuc, is toggling between `use only shmem` vs `use only register`. But this PR is updating it to extend persistent buffer to both shmem and register.